src/main/resources/db/migration/0300/V348__UpdateApplicationStatuses.sql

```diff
@@ -0,0 +1,33 @@
+ALTER TABLE accelerator.applications
+    DROP CONSTRAINT applications_application_status_id_fkey;
+
+DELETE FROM accelerator.application_statuses;
+
+INSERT INTO accelerator.application_statuses (id, name)
+VALUES (1, 'Not Submitted'),
+       (2, 'Failed Pre-screen'),
+       (3, 'Passed Pre-screen'),
+       (4, 'Submitted'),
+       (5, 'Sourcing Team Review'), -- Renamed from 'PL Review'
+       (6, 'GIS Assessment'), -- Renamed from 'Ready for Review'
+       (7, 'Expert Review'), -- Combined 'Pre-check', 'Needs Follow-up'
+       (8, 'Carbon Assessment'), -- New status.
+       (9, 'P0 Eligible'), -- Renamed from 'Carbon Eligible'
+       (10, 'Accepted'),
+       (11, 'Issue Reassessment'), -- Combined 'Issue Active', 'Issue Pending', 'Issue Resolved'
+       (12, 'Not Eligible'); -- Renamed from 'Not Accepted'
+
+UPDATE accelerator.applications
+SET application_status_id = CASE
+    -- 7 -> 7, 'Pre-check' -> 'Expert Review'
+    WHEN application_status_id = 8 THEN 7 -- 'Needs Follow-up' -> 'Expert Review'
+    -- 11 -> 11, 'Issue Active' -> 'Issue Reassessment'
+    WHEN application_status_id = 12 THEN 11 -- 'Issue Pending' -> 'Issue Reassessment'
+    WHEN application_status_id = 13 THEN 11 -- 'Issue Resolved' -> 'Issue Reassessment'
+    WHEN application_status_id = 14 THEN 12 -- 'Not Accepted' -> 'Not Accepted'
+    ELSE application_status_id
+END;
+
+ALTER TABLE accelerator.applications
+    ADD CONSTRAINT applications_application_status_id_fkey
+        FOREIGN KEY (application_status_id) REFERENCES accelerator.application_statuses(id);
```

Comment:
Doing it this way works, but another option would be to do the `UPDATE` and then delete just IDs 13 and 14 from `application_statuses`. Then the foreign-key constraint can stay in place the whole time.